### PR TITLE
Add Mario & Luigi Superstar Saga

### PR DESCRIPTION
**Pull request by greggameplayer**
Adds Mario & Luigi Superstar Saga.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/mario-luigi-superstar-saga/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Mario & Luigi Superstar Saga"
+description = "It's the first opus of the series of \"Mario & Luigi\" remake for 3ds with a new little story add to the previous."
```
Comment (Hexagon12) [inline games/mario-luigi-superstar-saga/game.dat]: I don't like this description. Get something from Wikipedia instead.

File: games/mario-luigi-superstar-saga/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
+title = "Mario & Luigi Superstar Saga"
```
Comment (Hexagon12) [inline games/mario-luigi-superstar-saga/game.dat]: That's not the full title of the game. It should be _Mario & Luigi: Superstar Saga + Bowser's Minions_.

File: games/mario-luigi-superstar-saga/game.dat [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+release_date = "2017-09-06"
+
+[[testcases]]
+title = "00040000001B9000"
+compatibility = "1"
+date = "2017-11-13"
+version = "HEAD-8102cbf"
+cpu = "AMD FX 6300"
+gpu = "GeForce GTX 960"
+os = "Windows 7"
```
Comment (Hexagon12) [inline games/mario-luigi-superstar-saga/game.dat]: _Windows 7 (your version number)_

File: games/mario-luigi-superstar-saga/game.dat [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+title = "00040000001B9000"
+region = "eur"
+release_date = "2017-09-06"
+
+[[testcases]]
+title = "00040000001B9000"
+compatibility = "1"
+date = "2017-11-13"
+version = "HEAD-8102cbf"
+cpu = "AMD FX 6300"
```
Comment (Hexagon12) [inline games/mario-luigi-superstar-saga/game.dat]: _AMD FX-6300_

**Comment by Leo626** [comment]
Hmm? How did I not see this? Thanks for the submission but the game is already in the game compatibility database therefore this will be closed.

**Pull request closed without merging**